jwst/associations/lib/rules_level3_base.py

```diff
@@ -956,8 +956,17 @@ def get_exposure_type(self, item, default='science'):
         Returns
         -------
         exposure_type : 'science'
-            Always returns as science
+            Returns as science for most Exposures
+        exposure_type : 'target_acquisition'
+            Returns target_acquisition for mir_tacq
         """
+        try:
+            exp_type = item['exp_type']
+            if exp_type =='mir_tacq':
```

Comment:
A new MIRI exp_type that's in the process of being implemented is 'mir_taconfirm', so we should go ahead and include that value here too, in order to make this future-proof. We already allow for it in our data model schemas.

Reply:
Another general question: all that's being checked for here is the specific 'mir_tacq' value. Target acq's from all instruments should be treated the same way, so is there some other rule(s) somewhere that handles those (e.g. nrc_tacq, nis_tacq, nrs_wata, ...)

Reply:
The code now checks in any of the exp_types are in the ACQ_EXP_TYPES and if so lists them in the association as target_acquisition. This list includes,     
    'mir_tacq',
    'mir_taconfirm',
    'nis_taconfirm',
    'nis_tacq',
    'nrc_taconfirm',
    'nrc_tacq',
    'nrs_confirm',
    'nrs_msata',
    'nrs_taconfirm',
    'nrs_tacq',
    'nrs_taslit',
    'nrs_wata',